src/web/pages/App.tsx

```diff
@@ -1,27 +1,41 @@
-import './App.css';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import HomePage from './HomePage';
+import FinOpsHubsPage from './FinOpsHubsPage';
+import FinOpsWorkbooksPage from './FinOpsWorkbooksPage';
+import GovernanceWorkbookPage from './GovernanceWorkbookPage';
+import AzureOptimizationEnginePage from './AzureOptimizationEnginePage';
+import PowerShellModulePage from './PowerShellModulePage';
+import BicepRegistryModulesPage from './BicepRegistryModulesPage';
+import OpenDataPage from './OpenDataPage';
+import {PowerBIReportsPage} from './PowerBIReportsPage';
+import CostOptimizationWorkbookPage from './CostOptimizationWorkbookPage';
+import {SamplePage} from './SamplePage';
+import FluentUIProvider from '../components/FluentUIProvider'; // Use your FluentUIProvider
```

Comment:
Can it be that the FluentUIProvider in the components folder was not committed? Its not in this path

Reply:
This is okay now, pushed the latest changes